doc/user-guides/auth/anonymous-access.md

```diff
@@ -0,0 +1,159 @@
+# Enforcing anonymous access with Kuadrant AuthPolicy
+
+Learn how to allow anonymous access to certain endpoints using Kuadrant's `AuthPolicy`
+
+## Prerequisites
+
+You have installed Kuadrant in a [kubernetes](https://docs.kuadrant.io/latest/kuadrant-operator/doc/install/install-kubernetes/) or [OpenShift](https://docs.kuadrant.io/latest/kuadrant-operator/doc/install/install-openshift/) cluster.
+
+## Run the guide ① → ⑦
```

Comment:
There was a pr merged to remove the unicode numbers to make things easier and more unified as some docs had them and some docs didn't

Reply:
Oh, cool, wasn't aware of that. I'll update 👍